### PR TITLE
Feature/extend GitHub action

### PR DESCRIPTION
**Pull request by StephanKa**

### CODE REVIEW

File: .github/workflows/build_cmake.yml [hunk 1]
```diff
@@ -69,4 +78,4 @@ jobs:
       shell: bash
       # Execute tests defined by the CMake configuration.
       # See https://cmake.org/cmake/help/latest/manual/ctest.1.html for more detail
-      run: ctest -C $BUILD_TYPE
+      run: ctest -C $BUILD_TYPE
```
Comment (arnemertz) [inline .github/workflows/build_cmake.yml]: why remove the newline?

File: README.md [hunk 2]
```diff
@@ -7,20 +7,6 @@ With this project, we want to provide an example and starting point for C++ proj
 The project has initially been forked/copied from [Jason Turner's cpp_starter_project](https://github.com/lefticus/cpp_starter_project).
 
 
-<hr>
```
Comment (arnemertz) [inline README.md]: This change does not appear to be part of the GitHub Action change. Also, removing the note that this was the original README makes it appear as though it is our README while some sections still have references to the GUI libraries and links to lefticus' project. Shall we leave the README changes to a separate PR where we rework everything thoroughly?

**Review by arnemertz — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (StephanKa) [reply README.md]: I will remove this file from commit and then this should be solved on a different PR

File: .github/workflows/build_cmake.yml (hunk 1 above)
Comment (StephanKa) [reply .github/workflows/build_cmake.yml]: reverted

**Review by arnemertz — APPROVED** [review]
